tests/e2e/config_secret_utils.go

```diff
@@ -556,31 +559,52 @@ roles and privileges to test user
 func createTestUserAndAssignRolesPrivileges(masterIp string, configSecretTestUser string,
 	configSecretTestUserPassword string, configSecretTestUserAlias string, propagateVal string,
 	dataCenters []*object.Datacenter, clusters []string, hosts []string,
-	vms []string, datastores []string) {
+	vms []string, datastores []string, testUserOpToPerform string, testUserRolesOpToPerform string) {
 	roleMap := userRoleMap()
+	switch testUserOpToPerform {
+	case "createUser":
+		err := createTestUser(masterIp, configSecretTestUser, configSecretTestUserPassword)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred(), "couldn't execute command on host: %v , error: %s",
+			masterIp, err)
+	case "reuseUser":
+		framework.Logf("Test user already exist")
+	default:
+		err := deleteTestUser(masterIp, configSecretTestUser)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred(), "couldn't execute command on host: %v , error: %s",
+			masterIp, err)
+		err = createTestUser(masterIp, configSecretTestUser, configSecretTestUserPassword)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred(), "couldn't execute command on host: %v , error: %s",
+			masterIp, err)
+	}
 
-	framework.Logf("Create TestUser")
-	err := createTestUser(masterIp, configSecretTestUser, configSecretTestUserPassword)
-	gomega.Expect(err).NotTo(gomega.HaveOccurred(), "couldn't execute command on host: %v , error: %s",
-		masterIp, err)
-
-	framework.Logf("Create roles for TestUser")
-	err = createRolesForTestUser(masterIp, configSecretTestUser)
-	gomega.Expect(err).NotTo(gomega.HaveOccurred(), "couldn't execute command on host: %v , error: %s",
-		masterIp, err)
+	switch testUserRolesOpToPerform {
+	case "createRoles":
+		err := createRolesForTestUser(masterIp, configSecretTestUser)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred(), "couldn't execute command on host: %v , error: %s",
+			masterIp, err)
+	case "reuseRoles":
+		framework.Logf("Roles for testuser already exist")
+	default:
```

Comment:
you can use `case "recreate"` instead of `default`

Reply:
done